sdcm/cluster_k8s/__init__.py

```diff
@@ -2670,9 +2670,7 @@ def decommission(self, node: BaseScyllaPodContainer, timeout: int | float = None
         # node deletion using "terminate_node" command.
         scylla_shards = node.scylla_shards
 
-        # NOTE: on k8s we treat soft_timeout as the "hard" timeout,
-        # at least until we'll have case with bigger data sets in it
-        timeout = timeout or soft_timeout or (node.pod_terminate_timeout * 60)
+        timeout = timeout or (node.pod_terminate_timeout * 60)
```

Comment:
Try to add the `adaptive_timeout` also in k8s case

Reply:
done